src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGetInstalledPackage.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio.Contracts
+{
+    /// <summary>Information about an installed package</summary>
+    /// <remarks>To create an instance, use <see cref="NuGetContractsFactory.CreateNuGetInstalledPackage"/>.</remarks>
+    public sealed class NuGetInstalledPackage
```

Comment:
Sounds like we'll add transitive later on. 
When we merge this, please make sure that the follow up issues are created.

Reply:
Previously I was planning on adding it later, since it wouldn't be a protocol breaking change to add. However, then the class contructor/factory would need an additional parameter, and then we'll have binary compat concerns, so maybe it's best to add now, even though we'll only have direct dependencies initially.

Reply:
I think it's ok for the type to contain that information.